app/pages/admin/project-status.cjsx

```diff
@@ -101,13 +101,16 @@ ProjectExperimentalFeatures = React.createClass
     @props.project.update({experimental_tools: tools})
 
   render: ->
-    <div>
+    <div className="project-status__section">
+      <h4>Experimental Features</h4>
       <AutoSave resource={@props.project}>
-        {EXPERIMENTAL_FEATURES.map (task) =>
-          <label key={task}>
-            <input type="checkbox" name={task} checked={@setting(task)} onChange={@updateTasks.bind @, task} />
-            {task.charAt(0).toUpperCase() + task.slice(1)}
-          </label>}
+        <div className="project-status__section--table">
```

Comment:
Sorry, I don't want to nitpick, this is a new way of doing things and I want to make sure we' start out on the same page.

Should this be the element `section-table`? This implies it's another nested `section` with a `table` modifier.